packages/core/src/schema.ts

```diff
@@ -1,3 +1,4 @@
 export function createSchema<T>(schema?: any): T {
```

Comment:
```suggestion
/**
 * @deprecated Use `t<SomeType>()` instead.
 **/
export function createSchema<T>(schema?: any): T {
```

Reply:
Will this work as intended? Since t is an alias it might rely on the createSchema TSDocs

Reply:
Maybe move the function definition to `t` instead?

This is nit-level but useful for folks using `createSchema`; can be done in a separate PR by the way